lua/hbac/setup.lua

```diff
@@ -2,12 +2,21 @@ local state = require("hbac.state")
 
 local M = {
 	opts = {
-		threshold = 10,
 		autoclose = true,
+		threshold = 10,
 		close_buffers_with_windows = false,
 		close_command = function(bufnr)
 			vim.api.nvim_buf_delete(bufnr, {})
 		end,
+		telescope = {
+			mappings = {
```

Comment:
This is not clear about normal and insert mappings (I had to see the map function to determine it's `<action> = {<insert_mapping>, <normal_mapping>}`). Can we do it like Telescope's own configuration table?

```lua
mappings = {
	n = {...}
	i = {...}
}
```

Reply:
Sure thing, I agree it's good to be consistent, I was just going for a more compressed opts table for the user, but I realize that's easier for me to visualize as the one writing it.

Reply:
I agree with ditsuke here. I think the most usual case will be that the user just leaves the table empty and the default mappings will be used. If they want to modify they will be most likely more familiar with the telescope way.

Reply:
So the opts could become:

```lua
  mappings = {
    n = {
      close_unpinned = "<M-c>",
      delete_buffer = "<M-x>",
      toggle_all = "<M-a>",
      toggle_selections = "<M-y>",
    },
    i = {
      close_unpinned = "<M-c>",
      delete_buffer = "<M-x>",
      toggle_all = "<M-a>",
      toggle_selections = "<M-y>",
    },
  },
```

and the mapping loop in the `attach_mappings` module would become (a much nicer looking)

```lua
for mode, hbac_cmds in pairs(hbac_config.telescope.mappings) do
  for hbac_cmd, key in pairs(hbac_cmds) do
    map(mode, key, hbac_telescope_actions[hbac_cmd])
  end
end
```

Reply:
Fine by me, I only use insert mode anyway 😅

Reply:
Updated in https://github.com/axkirillov/hbac.nvim/pull/5/commits/62ac2facb64f425f383a4a6068e8cededd39fc25

Reply:
Updated in https://github.com/axkirillov/hbac.nvim/pull/5/commits/62ac2facb64f425f383a4a6068e8cededd39fc25